app/features/dm.ts

```diff
@@ -180,6 +194,23 @@ export class DirectedMessageController implements DirectMessageGetter {
         return messages;
     }
 
+    public getLastestMessage(filter: "send_by_me" | "received_by_me"): ChatMessage | undefined {
```

Comment:
We don't need to separate senders in this interface

Reply:
ok